two_population_analysis/Snakefile

```diff
@@ -1,73 +1,172 @@
+    run:
+        dadi_utils.ts_to_dadi_sfs(input[0], output[0], sample_size = num_sampled_genomes_per_replicate)
 
 rule compare_msprime_sfs_to_OutOfAfrica_model:
-    input:
-        sfs_files
-    output:
-        "results_comparison/homo_sapiens_OutOfAfrica/compare_dadi_to_msprime_joint_sfs_resid_range_50.pdf"
-    run: dadi_utils.compare_msprime_dadi_OutOfAfrica(sfs_files,output[0])
+    input: expand(output_dir + "/{seeds}/{chrms}_dadi_joint_sfs.fs", seeds=seed_array, chrms=chrm_list)
+
+    output: output_dir +  "/{seeds}/dadi_analysis/compare_dadi_to_int_sfs_resid_range_50.pdf"
+    run:
+        inputs = expand(output_dir + "/{seeds}/{chrms}_dadi_joint_sfs.fs", seeds=wildcards.seeds, chrms=chrm_list)
+        dadi_utils.compare_msprime_dadi_OutOfAfrica(inputs,output[0])
 
 rule fit_dadi_model_to_msprime_sfs:
-    input:
-        sfs_files
+    input: expand(output_dir + "/{seeds}/{chrms}_dadi_joint_sfs.fs", seeds=seed_array, chrms=chrm_list)
     output:
-        "results_fits/homo_sapiens_OutOfAfrica/{demo_models}/model_fit_{fit_seeds}.pdf",
-        "results_fits/homo_sapiens_OutOfAfrica/{demo_models}/model_params_{fit_seeds}.txt"
+        output_dir +  "/{seeds}/dadi_analysis/{demo_models}/model_fit_{fit_seeds}.pdf",
+        output_dir +  "/{seeds}/dadi_analysis/{demo_models}/model_params_{fit_seeds}.txt"
+
     threads: 4
-    run: dadi_utils.fit_dadi_model(sfs_files,output[0],output[1],wildcards.demo_models, wildcards.fit_seeds)
+    run:
+        inputs = expand(output_dir + "/{seeds}/{chrms}_dadi_joint_sfs.fs", seeds=wildcards.seeds, chrms=chrm_list)
+        dadi_utils.fit_dadi_model(inputs,output[0],output[1],wildcards.demo_models, wildcards.fit_seeds)
+
+
+
+# ###############################################################################
+# fastsimcoal
+# ###############################################################################
+
+
+rule dadi_sfs_to_fsc:
+
+    # TODO: replace "IM" in the paths below with {demo_models}
+    # note that the dadi 2D SFSs mask the 0,0 entry which fsc uses - need to figure out how to unmask this entry
+    input: expand(output_dir + "/{seeds}/{chrms}_dadi_joint_sfs.fs", chrms=chrm_list,seeds=seed_array)
+
+    #note that fsc is very picky about file names
+    output: output_dir + "/{seeds}/fsc_analysis/IM_jointDAFpop1_0.obs"
+
+    threads: 2
+    run:
+        inputs = expand(output_dir + "/{seeds}/{chrms}_dadi_joint_sfs.fs", chrms=chrm_list,seeds=wildcards.seeds)
+        dadi_out_path = output_dir + "/" + wildcards.seeds + "/all_chrm_dadi_joint_sfs.fs"
+        fsc_out_path  = output_dir + "/" + wildcards.seeds + "/fsc_analysis/IM_jointDAFpop1_0.obs"
+        fastsimcoal.dadi_to_fsc_sfs(inputs, dadi_out_path, fsc_out_path, sample_size=num_sampled_genomes_per_replicate)
+
+
+rule fsc_setup:
+    input: expand(output_dir + "/{seeds}/fsc_analysis/IM_jointDAFpop1_0.obs", seeds=seed_array)
+
+    output: output_dir + "/{seeds}/fsc_analysis/IM.{file_type}"
```

Comment:
"file_type" wildcard is not defined. I would replace this line with these two:
```
162         output_dir + "/{seeds}/fsc_analysis/IM.est",
163         output_dir + "/{seeds}/fsc_analysis/IM.tpl"
```
or something equivilent